content/W2016/W16-event-code-party.md

```diff
@@ -0,0 +1,21 @@
+title: Code Party
+Date: 2016-04-02
+Category: Events
+Tags: code party, social, open source, academic
+Slug: code-party-w16
+Author: Anna Lorimer
+Summary: A casual day of fun and work!
+
+Come out for a fun day of coding with fellow women in CS! Work on that project
```

Comment:
Could we rephrase this? when we say women, we're excluding the non-binary individuals
Come out for a fun day of coding with WiCS!